### DISCUSSION TITLE
github.com with IP 140.82.121.3 not reachable

### DISCUSSION THREAD

**Discussion by mariusnauj** [community/community#66444] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello, soome pople have problem reaching github.com. Look at:

https://github.com/home-assistant/core/issues/99816
https://github.com/home-assistant/core/issues/99795

Hello guys, I did a lot of analysis and find out:

From my location [github.com](http://github.com/) response with 2 IP: 140.82.121.4 and 140.82.121.3, and from my home network I can’t ping 140.82.121.3. So when I reboot HA and at that time my network pointing [github.com](http://github.com/) to IP adress 140.82.121.4 HA starting fine. But if my network at this time pointing [github.com](http://github.com/) to IP 140.82.121.3 HA supervisor do not start up because supervisor can’t access [github.com](http://github.com/).
I tried pinging from my job both IP adreses responding.

Here it’s traceroutes:

Tracing route to [lb-140-82-121-4-fra.github.com](http://lb-140-82-121-4-fra.github.com/) [140.82.121.4]
over a maximum of 30 hops:

1 3 ms 3 ms 9 ms unifi.localdomain [192.168.8.1]
2 7 ms 10 ms 15 ms [78-61-43-254.static.zebra.lt](http://78-61-43-254.static.zebra.lt/) [78.61.43.254]
3 6 ms 5 ms 5 ms [82-135-179-128.static.zebra.lt](http://82-135-179-128.static.zebra.lt/) [82.135.179.128]
4 31 ms 28 ms 44 ms [xe-1-2-0.mpr1.fra4.de.above.net](http://xe-1-2-0.mpr1.fra4.de.above.net/) [80.81.194.26]
5 * 30 ms 29 ms [ae27.cs1.fra9.de.eth.zayo.com](http://ae27.cs1.fra9.de.eth.zayo.com/) [64.125.30.254]
6 36 ms 28 ms 32 ms [ae1.mcs1.fra9.de.eth.zayo.com](http://ae1.mcs1.fra9.de.eth.zayo.com/) [64.125.29.65]
7 32 ms 31 ms 30 ms 82.98.193.29
8 * * * Request timed out.
9 * * * Request timed out.
10 31 ms 31 ms 29 ms [lb-140-82-121-4-fra.github.com](http://lb-140-82-121-4-fra.github.com/) [140.82.121.4]

Trace complete.

C:\Users\marnau>tracert 140.82.121.3

Tracing route to [lb-140-82-121-3-fra.github.com](http://lb-140-82-121-3-fra.github.com/) [140.82.121.3]
over a maximum of 30 hops:

1 2 ms 2 ms 2 ms unifi.localdomain [192.168.8.1]
2 * * 4 ms [78-61-43-254.static.zebra.lt](http://78-61-43-254.static.zebra.lt/) [78.61.43.254]
3 6 ms 7 ms 4 ms [82-135-179-128.static.zebra.lt](http://82-135-179-128.static.zebra.lt/) [82.135.179.128]
4 29 ms 28 ms 28 ms [xe-1-2-0.mpr1.fra4.de.above.net](http://xe-1-2-0.mpr1.fra4.de.above.net/) [80.81.194.26]
5 * * * Request timed out.
6 33 ms 37 ms 36 ms [ae1.mcs1.fra9.de.eth.zayo.com](http://ae1.mcs1.fra9.de.eth.zayo.com/) [64.125.29.65]
7 33 ms 29 ms 29 ms 82.98.193.29
8 * * * Request timed out.
9 * * * Request timed out.
10 * * * Request timed out.
11 * * * Request timed out.
12 * * * Request timed out.
13 * * * Request timed out.
14 Transmit error: code 1232.

Trace complete.

So I think problem not HA update or internet provider, but problem is that github server with IP 140.82.121.3 drop packet from some IP ranges.

**Comment by dominykasn** [comment]
We are experiencing same issue:
```
traceroute to 140.82.121.6 (140.82.121.6), 64 hops max, 52 byte packets
 1  192.168.10.1 (192.168.10.1)  5.177 ms  2.953 ms  3.804 ms
 2  82-135-232-233.static.zebra.lt (82.135.232.233)  4.650 ms  3.774 ms  3.789 ms
 3  xe-1-2-0.mpr1.fra4.de.above.net (80.81.194.26)  27.217 ms  26.730 ms  26.834 ms
 4  ae27.cs1.fra9.de.eth.zayo.com (64.125.30.254)  29.116 ms  27.113 ms  27.048 ms
 5  ae1.mcs1.fra9.de.eth.zayo.com (64.125.29.65)  37.007 ms  27.498 ms  27.294 ms
 6  82.98.193.29.ipyx-270403-002-zyo.zip.zayo.com (82.98.193.29)  46.706 ms  31.431 ms  29.879 ms
 7  * * *
 8  * * *
 9  * * *
10  * * *
11  * * *
12  * * *
13  * * *
14  * * *
15  * * *
16  * * *
17  * * *
```

**Comment by mariusnauj** [comment]
Yes 140.82.121.6 too. From my home I can't ping, from my work it's pinging. I think it's github problem, they drop some Ip ranges.

**Comment by lukasauk** [comment]
yes, github please help

**Comment by mariusnauj** [comment]
140.82.121.3   140.82.121.5  140.82.121.6  not reachable from my home network.

**Comment by henrik-nil-acc** [comment]
We have the same issue, 140.82.121.3 and 140.82.121.4 not working properly.
I edited /etc/hosts and set github.com to the US ip: 140.82.112.4 and it works fine.

**Reply by marqu3z** [reply]
I can confirm this works for me too

**Reply by kpaaro** [reply]
Got it working thanks to this

**Reply by slavik-pastushenko** [reply]
Thank you!

**Reply by aPoluden** [reply]
Worked for us as well

**Reply by wydydq** [reply]
It works for me, thanks!

**Reply by sonichen** [reply]
thanks

**Reply by michaelchuang2023** [reply]
This works for me, cheers mate!

**Comment by mariusnauj** [comment]
@henrik-nil-acc how can I edit hosts file in HAOS, I edited but after restart it back to default ?

**Reply by henrik-nil-acc** [reply]
I don't know, I don't use HAOS.

**Reply by kpaaro** [reply]
Its probably setup like that, but if you have a local dns server like adguard and I think pihole, so you could setup a  DNS rewrite on the DNS server. @mariusnauj

**Reply by mariusnauj** [reply]
I dont have local DNS server, so I want to edit HAOS hosts file but I can't :)

**Comment by mariusnauj** [comment]
Maybe somebody know how to edit HAOS hosts file ?:)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @mariusnauj! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by RummsMurmel** [comment]
Hello,

same error since a few days.
140.82.121.4 is working - 140.82.121.3 always returns http return code 503.

i think it could just be a workaround to stick the ip address via hosts-file? there must be a source for this problem? any idea?
